### DISCUSSION TITLE
Remove ellipsis when using ClampLines()?

### DISCUSSION THREAD

**Discussion by austin-carr** [QuestPDF/QuestPDF#885] [Q&A] [answered]
Is there a way to remove the ellipsis that generate when text overflows a container that has a specific number of lines set via ClampLines()?

**Comment by MarcinZiabek** [comment]
Thank you for sharing your feedback. 

At this moment, QuestPDF does not support removing the ellipsis or replacing it with other glyphs. That being said, if there are more requests for this feature, I can prioritize it in the upcoming releases. What are your thoughts regarding the options and API?

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We apologize for the delayed response. Great news - this functionality is now available! You can clamp text lines with an ellipsis using the paragraph style API. You can find the documentation and examples here:
https://www.questpdf.com/api-reference/text/paragraph-style.html#clamp-line-with-ellipsis

We are closing this thread as the feature has been delivered. Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
